### PR TITLE
{compute} Bump up `azure-mgmg-compute` SDK to `26.1.0`

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
compute

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py [hunk 1]
```diff
@@ -744,14 +744,14 @@ def test_vm_create_by_attach_unmanaged_os_and_data_disks(self, resource_group):
     def test_vm_create_data_disk_delete_option(self, resource_group):
         self.cmd('vm create -n Delete_CLI1 -g {rg} --image RedHat:RHEL:7-RAW:7.4.2018010506 -l northeurope '
                  '--size Standard_E8as_v4 --generate-ssh-keys --public-ip-address "" --os-disk-size-gb 64 '
-                 '--data-disk-sizes-gb 200 --data-disk-delete-option Delete',
+                 '--data-disk-sizes-gb 200 --data-disk-delete-option Delete --admin-username vmtest',
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:747]: >  --admin-username vmtest

Why do we need to add this parameter to those tests?

Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:747]: If `--admin-user` is not specified, test will fail with `admin user name cannot contain upper case character A-Z, special characters \/"[]:|<>+=;,?*@#()! or start with $ or -`

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:747]: Why didn't this problem happen before?

Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:747]: If `admin-username` is not specified, the local machine name will be used as input by default. Specifying `admin-username` ensures that it meets the conditions (`admin user name cannot contain upper case character A-Z, special characters \/"[]:|<>+=;,?*@#()! or start with $ or -`).

**Pull request merged**